contracts/z_mocks/emissions/MockVotiumBribe.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IVotiumBribe } from "../../interfaces/IVotiumBribe.sol";
+
+contract MockVotiumBribe is IVotiumBribe{
+    using SafeERC20 for IERC20;
+
+    address public feeAddress = 0xe39b8617D571CEe5e75e1EC6B2bb40DdC8CF6Fa3; // Votium multisig  
+    event Bribed(address _token, uint256 _amount, bytes32 indexed _proposal, uint256 _choiceIndex);
+
+    function depositBribe(address _token, uint256 _amount, bytes32 _proposal, uint256 _choiceIndex) override external {
+        uint256 fee = 0;
+        uint256 bribeTotal = _amount- fee;
```

Comment:
it looks like the linter is not running against your staged code. Do you have Husky installed?
Or you can run `yarn lint:fix`

Reply:
Hi @naddison36 , I ran `yarn lint:fix` and `yarn lint`  to fix some warnings. I added some clean up